cmd/dispatcher/main.go

```diff
@@ -103,13 +103,14 @@ func main() {
 	rmqHelper := rabbit.NewRabbitMQHelper(1, make(chan bool), rabbit.DialWrapper)
 	defer rmqHelper.CleanupRabbitMQ(env.connection, logger)
 	for {
-		env.connection, env.channel, err = rmqHelper.SetupRabbitMQ(env.RabbitURL, rabbit.ChannelQoS, logger)
+		env.connection, env.channel, err = rmqHelper.SetupRabbitMQ(env.RabbitURL, rabbit.ChannelQoS, logger, true)
```

Comment:
Are we ever calling `SetupRabbitMQ` with false for listenToConnections?

Reply:
just for the source case (a patch before refactoring the rabbimtq setup pkg)